roles/services/tasks/fits.yml

```diff
@@ -15,10 +15,17 @@
   become: yes
   file: path={{ install_path }}/fits-{{ fits_ver }}/fits.sh mode=0755
 
+- name: add version to fits wrapper
```

Comment:
I think you can combine adding the version number and copying the file in a single task.